hca/util/__init__.py

```diff
@@ -302,6 +303,47 @@ def login(self, access_token=""):
                                         token_type="Bearer")
         print("Storing access credentials")
 
+    def _atomic_write_to_file(self, file, dir, content):
+        """
+        Atomically writes content a to file.
+
+        Creates a temp file and renames file (atomic operation) to desired filename.
+        To ensure atomicity, the src and dest dirs must be the same.
+
+        :param file: File name only (e.g. swagger.json)
+        :param dir: Shared directory of temp and dest file
+        :param content: Content to write to file
+        """
+        temp_filename = ""
```

Comment:
do you actually need this?

Reply:
Not necessarily, but it clears an IDE warning (and possibly linter?)